logrus/logrusentry.go

```diff
@@ -121,7 +121,6 @@ func (h *Hook) Fire(entry *logrus.Entry) error {
 		if h.fallback != nil {
 			return h.fallback(entry)
 		}
-		return errors.New("failed to send to sentry")
```

Comment:
I'm questioning the usefulness of handling the return value of `CaptureEvent` altogether. By default, the ID is always returned, even if the `HTTPTransport` does not successfully submit the event to Sentry, as we do not have a back channel to the main thread. So this only applies when using the `HTTPSyncTransport`.